source/content/drupal-9.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Drupal 9 Early Access
+description: Test the latest Drupal on the Pantheon Website Management Platform.
+categories: [get-started]
+tags: [site]
+earlyaccess: true
+contributors: [populist, edwardangert]
+reviewed: "2020-05-21"
+---
+
+Drupal 9 is, according to [drupal.org](https://www.drupal.org/docs/understanding-drupal/drupal-9-resources), "a cleaned-up version of Drupal 8... \[with] deprecated code removed and third-party dependencies updated."
+
+Drupal 9 updates Drupal’s underlying dependencies like [Symfony 4.4](https://symfony.com/releases/4.4) and [Twig 2](https://twig.symfony.com/doc/2.x/index.html), removes several deprecated API functions in favor of better options, and allows everyone running Drupal 8.8+ an easy upgrade path to Drupal 9 & beyond.
+
+<Alert title="Warning" type="danger">
+
+**Do not use this, or any, Early Access release for your live site.**
+
+Currently, Drupal 9 is only available on Pantheon as an Early Access feature release. This means the feature is currently in active development and is intended for testing and evaluation purposes only.
+
+Pantheon engineers are rolling out changes often, and easy updates from Drupal 9 Early Access are not guaranteed.
```

Comment:
suggest deleting this sentence

Reply:
I'm hesitant to remove this as it's a soft warning that users might not be able to update once they're on D9. Given the other warnings up top, I don't think we _have_ to keep it, but I like it as a reminder that folks shouldn't expect updates to definitely work.

leaving for copy editor